### PR TITLE
QA Observation Bugfix/FOUR-12964: There is no option to clear the line input of search in Process Browser

### PR DESCRIPTION
**Pull request by CarliPinell**
## How to Test
-  Have a process
1. Go to Launch Pad
2. Click on category with processes
3. Check the search engine
4. Check the "X" icon to clear the input search

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-12964

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-cd050926ee.eng.processmaker.net

File: resources/js/processes-catalogue/components/utils/SearchCards.vue
Comment (fagubla) [inline resources/js/processes-catalogue/components/utils/SearchCards.vue:1]: review this documentation
https://bootstrap-vue.org/docs/components/input-group

```javascript
<b-input-group-append>
    <b-button variant="outline-success">Button</b-button>
</b-input-group-append>
```

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes-catalogue/components/utils/SearchCards.vue
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/utils/SearchCards.vue:1]: The suggested code was applied

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-cd050926ee.eng.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Pull request merged**